docker/official/remco/templates/jaas-loginmodule.conf

```
@@ -22,36 +22,36 @@
         reportStatistics="{{ getv("/rundeck/jaas/ldap/reportstatistics", "true") }}"
-    {% if exists("/rundeck/jaas/ldap/roleusernamememberattribute") -%}
+{% if exists("/rundeck/jaas/ldap/roleusernamememberattribute") %}
         roleUsernameMemberAttribute="{{ getv("/rundeck/jaas/ldap/roleusernamememberattribute") }}"
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/ignoreroles") -%}
-        ignoreRoles={{ getv("/rundeck/jaas/ldap/ignoreroles") }}
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/storepass") -%}
-        storePass={{ getv("/rundeck/jaas/ldap/storepass") }}
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/clearpass") -%}
-        clearPass={{ getv("/rundeck/jaas/ldap/clearpass") }}
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/usefirstpass") -%}
-        useFirstPass={{ getv("/rundeck/jaas/ldap/usefirstpass") }}
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/tryfirstpass") -%}
-        tryFirstPass={{ getv("/rundeck/jaas/ldap/tryfirstpass") }}
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/userlastnameattribute") -%}
-        userLastNameAttribute={{ getv("/rundeck/jaas/ldap/userlastnameattribute") }}
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/userfirstnameattribute") -%}
-        userFirstNameAttribute={{ getv("/rundeck/jaas/ldap/userfirstnameattribute") }}
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/useremailattribute") -%}
-        userEmailAttribute={{ getv("/rundeck/jaas/ldap/useremailattribute") }}
-    {% endif %}
-    {% if exists("/rundeck/jaas/ldap/nestedgroups") -%}
-        nestedGroups={{ getv("/rundeck/jaas/ldap/nestedgroups") }}
-    {% endif %}
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/ignoreroles") %}
+        ignoreRoles="{{ getv("/rundeck/jaas/ldap/ignoreroles") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/storepass") %}
+        storePass="{{ getv("/rundeck/jaas/ldap/storepass") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/clearpass") %}
+        clearPass="{{ getv("/rundeck/jaas/ldap/clearpass") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/usefirstpass") %}
+        useFirstPass="{{ getv("/rundeck/jaas/ldap/usefirstpass") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/tryfirstpass") %}
+        tryFirstPass="{{ getv("/rundeck/jaas/ldap/tryfirstpass") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/userlastnameattribute") %}
+        userLastNameAttribute="{{ getv("/rundeck/jaas/ldap/userlastnameattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/userfirstnameattribute") %}
+        userFirstNameAttribute="{{ getv("/rundeck/jaas/ldap/userfirstnameattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/useremailattribute") %}
+        userEmailAttribute="{{ getv("/rundeck/jaas/ldap/useremailattribute") }}"
+{% endif %}
+{% if exists("/rundeck/jaas/ldap/nestedgroups") -%}
+        nestedGroups="{{ getv("/rundeck/jaas/ldap/nestedGroups") }}"
```

was this case change intentional?
```suggestion
        nestedGroups="{{ getv("/rundeck/jaas/ldap/nestedgroups") }}"
```

---

Yes. remco was failing to render when RUNDECK_JAAS_LDAP_NESTEDGROUP was set with the mixed case in getv.